### ISSUE TITLE
Tall R wave

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10839]
**Preferred term label:**
Tall R wave

**Synonyms**

**Definition (free text, please give PubMed ID)**

https://www.ncbi.nlm.nih.gov/medgen/?term=C0429093
Prominent r-wave in ECG lead V1 (or lead V2) with elevated R/S ratio. Can by seen in right ventricular hypertrophy (RVH) and other cardiovascular conditions.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Becker muscular dystrophy
Duchenne muscular dystrophy

**Your nano-attribution (ORCID)**
